main.php

```diff
@@ -70,229 +76,152 @@ function rg4wp_js()
     printf($script, get_option('rg4wp_apikey'), get_bloginfo('version'));
 }
 
-function load_jquery()
-{
-    wp_enqueue_script('jquery');
-}
+if (
+    1 == get_option('rg4wp_status')
+    && !rg4wp_isIgnoredDomain()
+    && get_option('rg4wp_apikey')
+    && !(1 == get_option('rg4wp_noadmintracking', 0) && is_admin())
+) {
+    RaygunClientManager::getInstance()->SetVersion(get_bloginfo('version'));
+
+    function getErrorTag(int $errno): string {
+        $errConsts = [
```

Comment:
Ideally, you want your constants, settings, strings, etc., on a separate file from the code